### PR TITLE
Updating stylesheet with color variables

### PR DESCRIPTION
**Pull request by jegelstaff**

### CODE REVIEW

**Comment by jegelstaff** [comment]
This will make it a lot easier to make simple new themes, just with different colours. One example: https://gvine.ca

File: themes/Anari/css/style.css [hunk 1]
```diff
@@ -10,6 +10,12 @@ body {
   line-height: 1.4;
 }
 
+:root {
```
Comment (arvinsingla) [inline themes/Anari/css/style.css:13]: Can we move this to after the meyer reset. Ideally the reset should happen first and then our custom styling afterward.

**Review by arvinsingla — APPROVED** [review]
Just one minor comment that I think should be addressed. Otherwise LG2M!

File: themes/Anari/css/style.css (hunk 1 above)
Comment (jegelstaff) [reply themes/Anari/css/style.css:13]: Yes!

**Pull request merged**